README.md

```diff
@@ -58,7 +60,46 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
         https://localhost/
 
-Done!
+#### Semi Dockerized Setup (continuation from General Setup 1-7)
```

Comment:
Could we maybe add a one or two sentence comment on why someone would prefer to do this instead of the fully Dockerized setup?

Reply:
++ Docs all the way